index.bs

```diff
@@ -217,13 +213,6 @@ dictionary OperandDescriptor {
   // The dimensions field is only required for tensor operands.
```

Comment:
As we are going to remove the `tensor-type` values of `OperandType`, we may need to define how to define a scalar. The options are no `dimensions` member, e.g. `{type: 'float32'}` or empty `dimensions`, e.g. {type: 'float32', dimensions: []}.

Reply:
I plan to open a new issue on scalar tensor.

Reply:
+1